examples/tfengine/app_foundation.hcl

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This example is still under development and is not considered stable!
+# Breaking changes to components for this example will not be part of releases.
+
+template "app" {
+  recipe_path = "../../templates/tfengine/recipes/app_foundation.hcl"
+  data = {
+    constants = {
+      shared = {
+        env_code            = "s"
+        folder_id           = "123"
+        billing_account     = "000"
+        project_prefix      = "example-prefix"
+        state_bucket        = "example-state"
+        bigquery_location   = "us-east1" # No BigQuery region in us-central1.
+        cloud_sql_region    = "us-central1"
+        cloud_sql_zone      = "a"
+        compute_region      = "us-central1"
+        gke_region          = "us-central1"
+        storage_location    = "us-central1"
+        secret_locations    = ["us-central1"]
```

Comment:
Why is this plural while the others are singular? What does it mean to have multiple secret_locations?